configure.ac

```diff
@@ -1031,6 +1036,7 @@ echo "Gcc            :  \"$BUILD_GCC\""
 AS_IF([test x$with_lustre != xno],[
 echo "Linux kernel   :  \"$LINUX_OBJ\""
 echo "Lustre         :  \"$LUSTRE\""
+echo "ISAL_LIBS      :  \"$ISAL_LIBS\""
```

Comment:
This ISAL has nothing to do with "with_lustre".
Please remove it out of this checking.
Print it with others, e.g. GALOIS_LIBS

Reply:
Please test:
If there is no "isa-l.h" present under "/usr/include", the "configure.ac" still succeeds, and the "ISAL_LIBS" is still set to "-lisal". 
(you can do this "mv /usr/include/isa-l.h /usr/include/isa-l.h.bak" and test"

Reply:
So, I propose to have:
```
diff --git a/configure.ac b/configure.ac
index 1957ee4..0bc3814 100644
--- a/configure.ac
+++ b/configure.ac
@@ -641,9 +641,10 @@ AC_SUBST([ISAL_LIBS])
 
 AC_CHECK_HEADERS([arpa/inet.h fcntl.h limits.h malloc.h memory.h netdb.h \
                   netinet/in.h stddef.h stdint.h stdlib.h string.h sys/ioctl.h \
-                  sys/socket.h sys/time.h unistd.h isa-l.h])
+                  sys/socket.h sys/time.h unistd.h])
 
 AC_CHECK_HEADERS([pthread.h], [], [AC_MSG_ERROR([pthread.h cannot be found!])])
+AC_CHECK_HEADERS([isa-l.h],   [], [AC_MSG_ERROR([isa-l.h cannot be found!])])
 
 
 AC_HEADER_STDBOOL
@@ -1031,12 +1032,12 @@ echo "PROFILER_LIBS  :  \"$PROFILER_LIBS\""
 echo "GALOIS_LIBS    :  \"$GALOIS_LIBS\""
 echo "YAML_LIBS      :  \"$YAML_LIBS\""
 echo "UUID_LIBS      :  \"$UUID_LIBS\""
+echo "ISAL_LIBS      :  \"$ISAL_LIBS\""
 echo ""
 echo "Gcc            :  \"$BUILD_GCC\""
 AS_IF([test x$with_lustre != xno],[
 echo "Linux kernel   :  \"$LINUX_OBJ\""
 echo "Lustre         :  \"$LUSTRE\""
-echo "ISAL_LIBS      :  \"$ISAL_LIBS\""
 ])
 echo ""
 

```
To have separate checking of AC_CHECK_HEADERS for isa-l.h, because this will report error if not found.

Reply:
So, I propose to have:
```
diff --git a/configure.ac b/configure.ac
index 1957ee4..0bc3814 100644
--- a/configure.ac
+++ b/configure.ac
@@ -641,9 +641,10 @@ AC_SUBST([ISAL_LIBS])
 
 AC_CHECK_HEADERS([arpa/inet.h fcntl.h limits.h malloc.h memory.h netdb.h \
                   netinet/in.h stddef.h stdint.h stdlib.h string.h sys/ioctl.h \
-                  sys/socket.h sys/time.h unistd.h isa-l.h])
+                  sys/socket.h sys/time.h unistd.h])
 
 AC_CHECK_HEADERS([pthread.h], [], [AC_MSG_ERROR([pthread.h cannot be found!])])
+AC_CHECK_HEADERS([isa-l.h],   [], [AC_MSG_ERROR([isa-l.h cannot be found!])])
 
 
 AC_HEADER_STDBOOL
@@ -1031,12 +1032,12 @@ echo "PROFILER_LIBS  :  \"$PROFILER_LIBS\""
 echo "GALOIS_LIBS    :  \"$GALOIS_LIBS\""
 echo "YAML_LIBS      :  \"$YAML_LIBS\""
 echo "UUID_LIBS      :  \"$UUID_LIBS\""
+echo "ISAL_LIBS      :  \"$ISAL_LIBS\""
 echo ""
 echo "Gcc            :  \"$BUILD_GCC\""
 AS_IF([test x$with_lustre != xno],[
 echo "Linux kernel   :  \"$LINUX_OBJ\""
 echo "Lustre         :  \"$LUSTRE\""
-echo "ISAL_LIBS      :  \"$ISAL_LIBS\""
 ])
 echo ""
 

```
To have separate checking of AC_CHECK_HEADERS for isa-l.h, because this will report error if not found.